website/docs/docs/build/environment-variables.md

```diff
@@ -109,13 +109,16 @@ The following environment variable is set automatically for the dbt Cloud IDE:
 
 Use case &mdash; This is useful in cases where you want to dynamically use the Git branch name as a prefix for a [development schema](/docs/build/custom-schemas) ( `{{ env_var ('DBT_CLOUD_GIT_BRANCH') }}` ).
 
-**dbt Cloud context**
+#### dbt Cloud context
 
-The following environment variables are set automatically for deployment runs:
+The following environment variables are set automatically: 
 
-- `DBT_ENV`: This key is reserved for the dbt Cloud application and will always resolve to 'prod'
+- `DBT_ENV`: This key is reserved for the dbt Cloud application and will always resolve to 'prod'. For deployment runs only.
```

Comment:
sugg using em dashes instead of colons. i'm a fan of using em dashes in a bulleted list. i think it sets the text off a bit better.

Reply:
Made an issue to do this in a follow-up PR to make it consistent across the page https://github.com/dbt-labs/docs.getdbt.com/issues/6103